sdk/formrecognizer/azure-ai-formrecognizer/src/main/java/com/azure/ai/formrecognizer/models/ModelTrainingStatus.java

```diff
@@ -0,0 +1,47 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.ai.formrecognizer.models;
+
+import com.azure.core.util.ExpandableStringEnum;
+
+import java.util.Collection;
+
+/**
+ * Defines enum values for ModelTrainingStatus.
+ */
+public final class ModelTrainingStatus extends ExpandableStringEnum<ModelTrainingStatus> {
```

Comment:
Isn't ModelStatus an implementation/model class and we don't want to be using those?

Reply:
`ModelStatus` is a generated enum which are generally fine to have in the public API (generated models are too but they need more hand holding sometimes). I think it would be better to move `ModelStatus` into public API and use it instead of making a copy of it.